docs/user_guide/HOWTO/Using_the_SIMP_Platform_RPM.rst

```diff
@@ -0,0 +1,133 @@
+.. _howto-use-the-simp-platform-rpm:
+
+===========================
+SIMP Platform RPM
+===========================
+
+Download Details
+---------------------------------
+
+Location
+********
+
+The SIMP Platform RPM can be downloaded `HERE
+<simp-platform-1.0.0-0.noarch.rpm>`_.
```

Comment:
@brandonrdn Can we bump this to a `latest` package or just point to the directory on the download server where it exists so that we don't have to update the docs every time this package changes?

Also, please try not to use `HERE` links if possible.